Makefile

```diff
@@ -43,12 +43,26 @@ TOFILES=\
 
 HFILES=\
 	dat.h\
-	sd-daemon.h\
 
 ifeq ($(OS),linux)
    LDLIBS+=-lrt
 endif
 
+# systemd support can be configured via USE_SYSTEMD:
+#        no: disabled
+#       yes: enabled, build fails if libsystemd is not found
+# otherwise: enabled if libsystemd is found
+ifneq ($(USE_SYSTEMD),no)
+ifeq ($(shell $(PKG_CONFIG) --exists libsystemd && echo $$?),0)
+	LDLIBS+=$(shell $(PKG_CONFIG) --libs libsystemd)
+	CPPFLAGS+=-DHAVE_LIBSYSTEMD
+else
+ifeq ($(USE_SYSTEMD),yes)
+$(error USE_SYSTEMD is set to "$(USE_SYSTEMD)", but $(PKG_CONFIG) cannot find libsystemd)
```

Comment:
```suggestion
	$(error USE_SYSTEMD is set to "$(USE_SYSTEMD)", but $(PKG_CONFIG) cannot find libsystemd)
```

Reply:
Make doesn't seem to like a tab there unfortunately:

`Makefile:61: *** commands commence before first target.  Stop.`

Reply:
Oh, I forgot how picky the make is. Nevermind!